plugins/modules/aci_tenant_action_rule_profile.py

```diff
@@ -230,12 +318,36 @@ def main():
     )
 
     action_rule = module.params.get("action_rule")
+    set_community = module.params.get("set_community")
+    set_dampening = module.params.get("set_dampening")
+    set_next_hop = module.params.get("set_next_hop")
+    next_hop_propagation = module.params.get("next_hop_propagation")
+    multipath = module.params.get("multipath")
+    set_preference = module.params.get("set_preference")
+    set_metric = module.params.get("set_metric")
+    set_metric_type = MATCH_ACTION_RULE_SET_METRIC_TYPE_MAPPING.get(module.params.get("set_metric_type"))
```

Comment:
As mentioned in the comments in the constants file,  if needed default return of "" could be specified here.

```suggestion
    set_metric_type = MATCH_ACTION_RULE_SET_METRIC_TYPE_MAPPING.get(module.params.get("set_metric_type"), "")
```

Not sure if it is needed, assume None should be good enough.